test/step-function.localstack.test.ts

```diff
@@ -448,6 +457,114 @@ runtimeTestSuite<
     }
   );
 
+  test(
+    "object literal spread",
+    (parent) => {
+      return new StepFunction(parent, "sfn", async (input) => {
+        const literalValueObjectLiterals = {
+          x: 1,
+          y: "a",
+          j: { x: 2 },
+          ...input,
+        };
+
+        // the duplication tests depth of intrinsic
+        // TODO: after optimization is added, turn some of it
+        return {
+          ...{ a: 0, b: 2, c: 3 },
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...{ m: 0, n: 5, o: 6 },
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...input,
+          ...{ x: 7, y: 8, z: 9, a: 1, m: 4 },
```

Comment:
Do we want to support dynamic keys and values like we discussed with json format?

Reply:
No, I'm a bit nervous about the formatting with Json paths so wanted to be leave it as a separate task.

Reply:
What makes you nervous? Fine with it being a separate task - that's reasonable.

Reply:
Finding more limits in the size of a parameters, formatting issues, maybe it'll be fine. Need to write some tests.